From eb6cea27f23c0674cea47871f281aaffb8a8efc3 Mon Sep 17 00:00:00 2001
From: Elliott Sales de Andrade <quantum.analyst@gmail.com>
Date: Sun, 1 Jan 2017 04:06:56 -0500
Subject: [PATCH 4/9] Add #define to get printf-format specifiers.

This is needed on older compilers/stdlibs such as on Ubuntu Precise
which is used on Travis.
---
 src/timestamp.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/timestamp.h b/src/timestamp.h
index 5565154..6a7ccd0 100644
--- a/src/timestamp.h
+++ b/src/timestamp.h
@@ -18,6 +18,9 @@
 #ifdef _WIN32
 #include "win32port.h"
 #else
+#ifndef __STDC_FORMAT_MACROS
+#define __STDC_FORMAT_MACROS
+#endif
 #include <inttypes.h>
 #endif
 
-- 
2.14.1

